server/router/api.go

```diff
@@ -61,11 +61,18 @@ func apiRoutes(e *gin.RouterGroup) {
 					org.Use(session.MustOrgMember(true))
 					org.DELETE("", session.MustAdmin(), api.DeleteOrg)
 					org.GET("", api.GetOrg)
+
 					org.GET("/secrets", api.GetOrgSecretList)
 					org.POST("/secrets", api.PostOrgSecret)
 					org.GET("/secrets/:secret", api.GetOrgSecret)
 					org.PATCH("/secrets/:secret", api.PatchOrgSecret)
 					org.DELETE("/secrets/:secret", api.DeleteOrgSecret)
+
+					org.GET("/registry", api.GetOrgRegistryList)
```

Comment:
Most other endpoints  are using the plural version. Therefore I would use `registries` here as well.

Reply:
At first I started to use registries but later changed to registry as repo registry endpoint was already using `registry`

Reply:
@anbraten should I rename existing repo reigstry endpoints as well than?

Reply:
I would suggest we rename all to `/registries` and leave the `/registry` routes with a TODO to remove them in 3.x